tools/make/deploy.mk

```diff
@@ -85,7 +85,52 @@ deploy_for_e2e: _deploy-check-env-vars docker-build manifests _helm_setup kind-l
 		--set bindings.serviceLabels.label1="val1"
 
 .PHONY: deploy_multi_namespace
+## 1. We want to install the CRDs only once at the beginning
+## 2. We want to make a namespace-a and namespace-b
+## 3. We want to install the helm chart twice, watching only the namespace belonging to each one
 deploy_multi_namespace: _deploy-check-env-vars docker-build manifests _helm_setup kind-load-image ## Deploy multiple copies of the controller to the K8s cluster specified in ~/.kube/config.
+	helm upgrade ngrok-operator-crds $(CRD_CHART_DIR) --install \
+		--kube-context=kind-$(KIND_CLUSTER_NAME) \
+	    --namespace kube-system \
+		--create-namespace
+	kubectl --context=kind-$(KIND_CLUSTER_NAME) create namespace namespace-a || true
+	kubectl --context=kind-$(KIND_CLUSTER_NAME) create namespace namespace-b || true
```

Comment:
Nit: You could use `--create-namespace` instead of this similar to how the other make targets work